CONTRIBUTING.md

```diff
@@ -69,6 +69,12 @@ while __private__ roles are located in a separate subnet not visible to the outs
 Exocom let's services of all protection levels communicate.
 
 
+## Implementing new languages
+
+There are some early versions of exorelay written in haskall, java, and scala
```

Comment:
Typo in `haskall`, its spelled `Haskell`. Also, Java and Scala are names and should be uppercased.